persistence/src/main/java/io/nobt/persistence/nobt/NobtMapper.java

```diff
@@ -60,6 +72,10 @@ public NobtEntity mapToDatabaseModel(Nobt domainModel) {
         nobtEntity.setExplicitParticipant(domainModel.getParticipatingPersons());
 
         domainModel.getExpenses().stream().map(expenseMapper::mapToDatabaseModel).forEach(nobtEntity::addExpense);
+        domainModel.getDeletedExpenses().stream()
+                .map(expenseMapper::mapToDatabaseModel)
+                .peek(e -> e.setDeleted(true))
```

Comment:
Da wir eine Expense eigentlich nie mehr auf "nicht deleted" setzen wollen, könnte man in der Expense-Entity eine Methode anbieten die heißt `markDeleted()` oder so ähnlich und diesen boolean gar nie nach außen preisgibt. Dann kannst du hier auch ne Methodenreferenz verwenden. Kommt halt darauf an, wieviele solche "Convenience"-Methoden man in seiner Entity haben will.